models/marts/core/fct_ga4__sessions.sql

```diff
@@ -4,18 +4,20 @@ select
     client_key,
     session_key,
     stream_id,
+    user_id,
```

Comment:
In the case of a multi-day session, user_id could be null on day 1 and have a value on day 2. This would result in 2 rows with the same session_key. Should we take the max(user_id) here as well?